### PR TITLE
Browsing profiles and creating scans chapter

### PR DESCRIPTION
**Pull request by jhrozek**

### CODE REVIEW

File: workshop/content/exercises/03-creating-your-first-scan.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+Title: Scanning
+PrevPage: 02-installation
+NextPage: 04-remediations
+---
+
+This chapter would show you how to create, run and evaluate a compliance
```
Comment (JAORMX) [inline workshop/content/exercises/03-creating-your-first-scan.md]: s/would/will/

File: workshop/content/exercises/03-creating-your-first-scan.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+scan. Before actually starting a scan, we need to select a baseline we'll
+be scanning against.
+
+If you're familiar with OpenScap, you might be already familiar with how the
```
Comment (JAORMX) [inline workshop/content/exercises/03-creating-your-first-scan.md]: s/OpenScap/OpenSCAP/

**Review by JAORMX — CHANGES_REQUESTED** [review]

**Comment by jhrozek** [comment]
Suggestions fixed, no longer a WIP.

**Review by JAORMX — APPROVED** [review]
LGTM

**Pull request merged**